composer.json

```diff
@@ -13,7 +13,8 @@
             "tests\\Libero\\Browser\\": "tests/",
             "tests\\Libero\\ContentPageBundle\\": "vendor-extra/ContentPageBundle/tests/",
             "tests\\Libero\\JatsContentBundle\\": "vendor-extra/JatsContentBundle/tests/",
-            "tests\\Libero\\LiberoContentBundle\\": "vendor-extra/LiberoContentBundle/tests/"
+            "tests\\Libero\\LiberoContentBundle\\": "vendor-extra/LiberoContentBundle/tests/",
+            "tests\\Libero\\ViewsBundle\\": "vendor-extra/ViewsBundle/tests/"
```

Comment:
what is the direction of dependencies? Format-specific bundles depend on `ViewsBundle` where the interfaces are?

That module really needs a README to document common cases for adding/modifying Visitors.